website/content/maintainers/templates/governance-elections.md

```diff
@@ -8,6 +8,260 @@ aliases:
+and another example of this model is the [Knative governance].
+
+# Is This Template For Us?
+
+An election-based steering committee governance model is for you if your project:
+
+1) Is organized into distinct groups, like SIGs / WGs, that might have competing 
+priorities
+2) Wants to enforce employer representation limits for leadership
+3) Has a large number of maintainers (20+) who cannot easily govern themselves
+
+If you don’t yet meet these conditions, we recommend that you start with a
+simpler governance template, like the [Maintainer Council] template, and move
+to this template later when you actually need it. Starting with a
+complicated governance structure before you need it creates overhead and extra
+work for project members and maintainers whose time is often better spent on
+project development, rather than governance.
+
+Before deciding to implement this governance model for a CNCF project, please 
+reach out to [TAG Contributor Strategy] so that we can discuss how you might
+implement this for your CNCF project.
+
+## Requirements
+
+### What Do I Need To Know?
+
+There are quite a few things you need to know before creating a governance model
+of this complexity. Here is a partial list, but there are likely others depending
+on exactly how your project plans to operate:
+* A list of your current maintainers
+* Locations of your repositories and files
+* Details on your project's existing developer meetings
+* Details on mailing lists or other official communications channels, including both public and private channels for the Maintainers
+* Other committees that the Steering Committee may choose to delegate its authority to as-needed (security, code of conduct, etc).
+* Steering Committee meeting logistics - how often, when, where meeting notes will be published, etc.
+* Contact details for how community members can contact the Steering Committee
+* Composition - Number of seats, terms, etc.
+* How you will bootstrap the first steering committee
+* Eligibility criteria for members to run for seats on the Steering Committee
+* Election procedures for timing, election officers, eligibility of voters, etc.
+* Employer representation limits
+* Procedure for handling vacancies
+
+### What Do I Need to Customize?
+
+All sections of this template will likely need to be customized, so we recommend 
+that you reach out to [TAG Contributor Strategy] with questions about how to 
+customize it for your CNCF project.
+
+### What Else Is Required?
+
+This template assumes that you have already adopted the [Code of Conduct] and
+assumes that you are using the [Contributor Ladder], since it is a good way to provide
+guidance about eligibility requirements. It is possible to implement this without a
+[Contributor Ladder], but it is not recommended.
+
+## Glossary
+
+**Project**: Your entire CNCF project, rather than individual repositories or subprojects.
+
+**Steering Committee**: The overall governing body for your project.
```

Comment:
```suggestion
**Steering Committee**: The overall governing body for your project, and the list of maintainers by CNCF definition.

**Maintainers**:  the actual approvers/technical leadership of the project, which might be different from the membership of the Steering Committee. 

**CNCF Maintainers**: the list of "maintainers" per CNCF rules, as in people who are allowed to make decisions for the project that the CNCF will carry out.  In a Steering Committee project, this is the Steering Committee instead of the list of approvers.
```